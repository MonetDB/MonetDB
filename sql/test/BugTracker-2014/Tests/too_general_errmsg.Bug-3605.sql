start transaction;
CREATE TABLE "sys"."tt_20141016" (
"operatingday"          DATE,
"trip_id"               VARCHAR(16),
"pointorder"            SMALLINT,
"passagesequencenumber" SMALLINT,
"userstopcode"          VARCHAR(10),
"targetarrivaltime"     VARCHAR(8),
"targetdeparturetime"   VARCHAR(8),
"trip_hash"             BIGINT
);
create view tt as select * from tt_20141016;
CREATE TABLE "sys"."kv17_canceled" (
"server"                TIMESTAMP,
"messagetype"           VARCHAR(17),
"dataownercode"         VARCHAR(10),
"operatingday"          DATE,
"lineplanningnumber"    VARCHAR(10),
"journeynumber"         DECIMAL(6),
"reinforcementnumber"   DECIMAL(2),
"reasontype"            DECIMAL(3),
"subreasontype"         VARCHAR(10),
"reasoncontent"         VARCHAR(255),
"subreasoncontent"      VARCHAR(255),
"userstopcode"          VARCHAR(10),
"passagesequencenumber" DECIMAL(4),
"lagtime"               DECIMAL(4),
"targetarrivaltime"     TIME,
"targetdeparturetime"   TIME,
"journeystoptype"       VARCHAR(12),
"destinationcode"       VARCHAR(10),
"destinationname50"     VARCHAR(50),
"destinationname16"     VARCHAR(16),
"destinationdetail16"   VARCHAR(16),
"destinationdisplay16"  VARCHAR(16),
"trip_hash"             BIGINT,
"message"               TIMESTAMP
);
create view kv6 as select * from kv17_canceled; 
select operatingday, coalesce(gepubliceerd, 0) as gepubliceerd, coalesce(gereden, 0) as gereden, coalesce(geannuleerd, 0) as geannuleerd, coalesce(onbekend, 0) as onbekend, coalesce(extra, 0) as extra, coalesce(tochgezien, 0) as tochgezien from (select operatingday, count(trip_hash) as gepubliceerd from tt where pointorder = 1 group by operatingday) as a full outer join (select z.operatingday, count(*) as geannuleerd from (select trip_hash, operatingday from kv17_canceled except select trip_hash, operatingday from kv6 where kv6.messagetype = 'ARRIVAL') as u group by u.operatingday) as e using (operatingday) full outer join (select o.operatingday, count(trip_hash) as onbekend from (select distinct trip_hash, operatingday from tt) as o join (select distinct trip_hash from tt where (epoch(cast(tt.operatingday as timestamp with time zone)) + cast(split_part(tt.targetarrivaltime, ':', 1) as int) * 3600 + (cast(split_part(tt.targetarrivaltime, ':', 2) as int) + 10) * 60) < epoch(now()) except (select trip_hash from kv6 where messagetype = 'ARRIVAL' union all select trip_hash from kv17_canceled)) as p using (trip_hash) group by operatingday) as f using (operatingday) order by operatingday;
