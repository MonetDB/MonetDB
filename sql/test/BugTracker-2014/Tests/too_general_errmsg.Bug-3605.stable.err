stderr of test 'too_general_errmsg.Bug-3605` in directory 'sql/test/BugTracker-2014` itself:


# 09:47:06 >  
# 09:47:06 >  "mserver5" "--debug=10" "--set" "gdk_nr_threads=0" "--set" "mapi_open=true" "--set" "mapi_port=39443" "--set" "mapi_usock=/var/tmp/mtest-8689/.s.monetdb.39443" "--set" "monet_prompt=" "--forcemito" "--set" "mal_listing=2" "--dbpath=/home/niels/scratch/rc-clean/Linux-x86_64/var/MonetDB/mTests_sql_test_BugTracker-2014" "--set" "mal_listing=0" "--set" "embedded_r=yes"
# 09:47:06 >  

# builtin opt 	gdk_dbpath = /home/niels/scratch/rc-clean/Linux-x86_64/var/monetdb5/dbfarm/demo
# builtin opt 	gdk_debug = 0
# builtin opt 	gdk_vmtrim = no
# builtin opt 	monet_prompt = >
# builtin opt 	monet_daemon = no
# builtin opt 	mapi_port = 50000
# builtin opt 	mapi_open = false
# builtin opt 	mapi_autosense = false
# builtin opt 	sql_optimizer = default_pipe
# builtin opt 	sql_debug = 0
# cmdline opt 	gdk_nr_threads = 0
# cmdline opt 	mapi_open = true
# cmdline opt 	mapi_port = 39443
# cmdline opt 	mapi_usock = /var/tmp/mtest-8689/.s.monetdb.39443
# cmdline opt 	monet_prompt = 
# cmdline opt 	mal_listing = 2
# cmdline opt 	gdk_dbpath = /home/niels/scratch/rc-clean/Linux-x86_64/var/MonetDB/mTests_sql_test_BugTracker-2014
# cmdline opt 	mal_listing = 0
# cmdline opt 	embedded_r = yes
# cmdline opt 	gdk_debug = 536870922

# 09:47:07 >  
# 09:47:07 >  "mclient" "-lsql" "-ftest" "-Eutf-8" "-i" "-e" "--host=/var/tmp/mtest-8689" "--port=39443"
# 09:47:07 >  

MAPI  = (monetdb) /var/tmp/mtest-27483/.s.monetdb.35395
QUERY = select operatingday, coalesce(gepubliceerd, 0) as gepubliceerd, coalesce(gereden, 0) as gereden, coalesce(geannuleerd, 0) as geannuleerd, coalesce(onbekend, 0) as onbekend, coalesce(extra, 0) as extra, coalesce(tochgezien, 0) as tochgezien from (select operatingday, count(trip_hash) as gepubliceerd from tt where pointorder = 1 group by operatingday) as a full outer join (select z.operatingday, count(*) as geannuleerd from (select trip_hash, operatingday from kv17_canceled except select trip_hash, operatingday from kv6 where kv6.messagetype = 'ARRIVAL') as u group by u.operatingday) as e using (operatingday) full outer join (select o.operatingday, count(trip_hash) as onbekend from (select distinct trip_hash, operatingday from tt) as o join (select distinct trip_hash from tt where (epoch(cast(tt.operatingday as timestamp with time zone)) + cast(split_part(tt.targetarrivaltime, ':', 1) as int) * 3600 + (cast(split_part(tt.targetarrivaltime, ':', 2) as int) + 10) * 60) < epoch(now()) except (select trip_hash from kv6 where messagetype = 'ARRIVAL' union all select trip_hash from kv17_canceled)) as p using (trip_hash) group by operatingday) as f using (operatingday) order by operatingday;
ERROR = !SELECT: no such column 'z.operatingday'
CODE  = 42S22



# 09:47:07 >  
# 09:47:07 >  "Done."
# 09:47:07 >  

