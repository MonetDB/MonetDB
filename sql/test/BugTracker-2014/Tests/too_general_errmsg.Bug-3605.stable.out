stdout of test 'too_general_errmsg.Bug-3605` in directory 'sql/test/BugTracker-2014` itself:


# 09:47:06 >  
# 09:47:06 >  "mserver5" "--debug=10" "--set" "gdk_nr_threads=0" "--set" "mapi_open=true" "--set" "mapi_port=39443" "--set" "mapi_usock=/var/tmp/mtest-8689/.s.monetdb.39443" "--set" "monet_prompt=" "--forcemito" "--set" "mal_listing=2" "--dbpath=/home/niels/scratch/rc-clean/Linux-x86_64/var/MonetDB/mTests_sql_test_BugTracker-2014" "--set" "mal_listing=0" "--set" "embedded_r=yes"
# 09:47:06 >  

# MonetDB 5 server v11.19.4
# This is an unreleased version
# Serving database 'mTests_sql_test_BugTracker-2014', using 4 threads
# Compiled for x86_64-unknown-linux-gnu/64bit with 64bit OIDs dynamically linked
# Found 7.333 GiB available main-memory.
# Copyright (c) 1993-July 2008 CWI.
# Copyright (c) August 2008-2015 MonetDB B.V., all rights reserved
# Visit http://www.monetdb.org/ for further information
# Listening for connection requests on mapi:monetdb://localhost.nes.nl:39443/
# Listening for UNIX domain connection requests on mapi:monetdb:///var/tmp/mtest-8689/.s.monetdb.39443
# MonetDB/GIS module loaded
# MonetDB/SQL module loaded
# MonetDB/R   module loaded


# 09:47:07 >  
# 09:47:07 >  "mclient" "-lsql" "-ftest" "-Eutf-8" "-i" "-e" "--host=/var/tmp/mtest-8689" "--port=39443"
# 09:47:07 >  

#start transaction;
#CREATE TABLE "sys"."tt_20141016" (
#"operatingday"          DATE,
#"trip_id"               VARCHAR(16),
#"pointorder"            SMALLINT,
#"passagesequencenumber" SMALLINT,
#"userstopcode"          VARCHAR(10),
#"targetarrivaltime"     VARCHAR(8),
#"targetdeparturetime"   VARCHAR(8),
#"trip_hash"             BIGINT
#);
#create view tt as select * from tt_20141016;
#CREATE TABLE "sys"."kv17_canceled" (
#"server"                TIMESTAMP,
#"messagetype"           VARCHAR(17),
#"dataownercode"         VARCHAR(10),
#"operatingday"          DATE,
#"lineplanningnumber"    VARCHAR(10),
#"journeynumber"         DECIMAL(6),
#"reinforcementnumber"   DECIMAL(2),
#"reasontype"            DECIMAL(3),
#"subreasontype"         VARCHAR(10),
#"reasoncontent"         VARCHAR(255),
#"subreasoncontent"      VARCHAR(255),
#"userstopcode"          VARCHAR(10),
#"passagesequencenumber" DECIMAL(4),
#"lagtime"               DECIMAL(4),
#create view kv6 as select * from kv17_canceled; 

# 09:47:07 >  
# 09:47:07 >  "Done."
# 09:47:07 >  

